templates/data/incubator-buildpack-data.yaml

```diff
@@ -263,6 +263,29 @@ templates:
               pipeline.installer: reconciler
               pipeline.test: "fast-integration"
               pipeline.clusterprovisioning: reconciler
+          reconciler_image_autobump_jobConfig:
+            command: "/home/prow/go/src/github.com/kyma-incubator/reconciler/scripts/prow/reconciler-image-autobump.sh"
+          extra_refs_k8s-test-infra:
+            extra_refs:
+              k8s-test-infra:
+                - org: kubernetes
+                  repo: test-infra
+                  base_ref: master
+                  base_sha: "3fba7110c39a7a0e7fb087dbb4af0674cafd341c"
+                  path_alias: github.com/kubernetes/test-infra
+          github_token_mounts:
+            env:
+              GITHUB_LOGIN: "kyma-bot"
+              GITHUB_TOKEN: "/etc/github/token"
+              GIT_NAME: "Kyma Bot"
+              GIT_EMAIL: "kyma.bot@sap.com"
```

Comment:
Are those variables still needed? Those variables are provided inside the auto bump config YAML and I believe those were needed only when you were using `gh` CLI. As right now you are not using it then I see no sense in defining them again.

Reply:
I have removed them now.